### ISSUE TITLE
test - error on some systems with test import

### ISSUE THREAD

**Issue by tpurschke** [CactuseSecurity/firewall-orchestrator#1542]
the following error occurs reproducibly on test system, not on prod system:

```
TASK [test : make test import] *******************************************************************************************************************************************************
fatal: [fworch-srv]: FAILED! => changed=true
  cmd:
  - ./fworch-importer-single.pl
  - mgm_name=fortigate_test_khwkWdjlejojoRFRvlHYfKeCMRHRXXFn
  delta: '0:00:00.185500'
  end: '2022-02-08 08:45:54.705122'
  msg: non-zero return code
  rc: 2
  start: '2022-02-08 08:45:54.519622'
  stderr: |-
    DBD::Pg::st execute failed: ERROR:  duplicate key value violates unique constraint "import_control_only_one_null_stop_time_per_mgm_when_null"
    DETAIL:  Key (mgm_id)=(70) already exists. at CACTUS/FWORCH/import.pm line 342.
    DBD::Pg::st execute failed: ERROR:  current transaction is aborted, commands ignored until end of transaction block at CACTUS/FWORCH/import.pm line 345.
    DBD::Pg::st fetchrow failed: no statement executing at CACTUS/FWORCH/import.pm line 346.
    Use of uninitialized value $current_import_id in concatenation (.) or string at ./fworch-importer-single.pl line 100.
  stderr_lines: <omitted>
  stdout: |-
    version: 5ff, manufacturer: fortinet, current_import_id=ERR-set-import-lock-failed
    WARN-import-already-running: fortigate_test_khwkWdjlejojoRFRvlHYfKeCMRHRXXFn (ID: 70)
  stdout_lines: <omitted> 

```

**Comment by tpurschke** [comment]
both systems at a customer's are now affected

**Comment by tpurschke** [comment]
fixed by #1620

**Comment by tpurschke** [comment]
unfortunately the "TimeoutStopSec=300min" setting did not fix this error, we still get the same error message.
The wait for import to stop works though.

**Comment by alf-cactus** [comment]
Closed with #1628
